DataCollector/MessageDataCollector.php

```diff
@@ -60,6 +60,8 @@ public function collect(Request $request, Response $response, \Exception $except
                     ];
 
                     foreach ($logger->getMessages() as $message) {
+                        ($userContentTypeProp = new \ReflectionProperty(\Swift_Message::class, 'userContentType'))->setAccessible(true);
```

Comment:
Do we really need to use Reflection to access it ?

Reply:
I think so. It's available with `getContentType()` or in headers.. but only if you have no alternative parts; otherwise it becomes `multipart/alternative`.

Also the first boundary is not part of `message.children`. Im not aware of any further internals :)